executor/builder.go

```diff
@@ -824,7 +824,7 @@ func (b *executorBuilder) buildUnionScanFromReader(reader Executor, v *plannerco
 		us.dirty = GetDirtyDB(b.ctx).GetDirtyTable(physicalTableID)
 		us.conditions = v.Conditions
 		us.columns = x.columns
-		err = us.buildAndSortAddedRows(x.table)
```

Comment:
Add a TODO to remove `buildAndSortAddedRows`

Reply:
done.